src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/language/CognitiveServicesLanguageDetector.scala

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.language
+
+import net.liftweb.json
+
+import scalaj.http.Http
+
+case class CognitiveServicesLanguageDetectorAuth(key: String, apiUrlBase: String = "https://westus.api.cognitive.microsoft.com")
+
+@SerialVersionUID(100L)
+class CognitiveServicesLanguageDetector(
+  auth: CognitiveServicesLanguageDetectorAuth,
+  minConfidence: Double = 0.75
+) extends LanguageDetector {
+
+  def detectLanguage(text: String): Option[String] = {
+    if (text.isEmpty) {
+      return None
+    }
+
+    val textId = "0"
+    val requestBody = buildRequestBody(text, textId)
+    val response = callCognitiveServices(requestBody)
+    parseResponse(response, textId)
+  }
+
+  protected def callCognitiveServices(requestBody: String): String = {
```

Comment:
This is the old class that got renamed. I can delete it if you want, but given that we're keeping the Kafka sink around too, might as well keep this option around.